From 40193fc73ff72a61bc2d81db23d1f23a3baf37be Mon Sep 17 00:00:00 2001
From: Zineb El Azzouzi <zineb.elazzouzi@nxp.com>
Date: Tue, 3 Oct 2023 18:24:51 +0200
Subject: [PATCH 60/64] configs: arm64: imx93: add wakeupimx1 gpr mmu region to
 industrial and avb inmate cells Signed-off-by: Zineb El Azzouzi
 <zineb.elazzouzi@nxp.com>

---
 configs/arm64/imx93-harpoon-rtos.h | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/configs/arm64/imx93-harpoon-rtos.h b/configs/arm64/imx93-harpoon-rtos.h
index f912582c..7f4531a4 100644
--- a/configs/arm64/imx93-harpoon-rtos.h
+++ b/configs/arm64/imx93-harpoon-rtos.h
@@ -46,12 +46,14 @@ typedef enum IRQn {
 	MEM_REGION_RWX( 0x20480000, 0x20480000, KB(640)) /* OCRAM */
 
 #define RTOS_AVB_MEMORY_REGIONS \
-	RTOS_AUDIO_MEMORY_REGIONS,                                      \
-	MMIO_REGION_RW( 0x428a0000, 0x428a0000, KB(64)) /* ENET_Q*/
+	RTOS_AUDIO_MEMORY_REGIONS,                                     \
+	MMIO_REGION_RW( 0x428a0000, 0x428a0000, KB(64)) /* ENET_Q*/    \
+	MMIO_REGION_RWS( 0x42420000, 0x42420000, KB(64)) /* WAKEUPIMX1*/
 
 #define RTOS_INDUS_MEMORY_REGIONS \
 	MMIO_REGION_RW( 0x425b0000, 0x425b0000, KB(64)), /* FLEXCAN2*/  \
-	MMIO_REGION_RW( 0x428a0000, 0x428a0000, KB(64)) /* ENET_Q*/
+	MMIO_REGION_RW( 0x428a0000, 0x428a0000, KB(64)), /* ENET_Q*/    \
+	MMIO_REGION_RWS( 0x42420000, 0x42420000, KB(64)) /* WAKEUPIMX1*/
 
 #define RTOS_AUDIO_MEMORY_REGIONS \
 	MMIO_REGION_RW( 0x44340000, 0x44340000, KB(64))  /* LPI2C1 */
-- 
2.34.1

